toucan_connectors/mssql/mssql_connector.py

```diff
@@ -14,10 +14,68 @@ class MSSQLDataSource(ToucanDataSource):
         description='The name of the database you want to query. '
         "By default SQL Server selects the user's default database",
     )
+    table: constr(min_length=1) = Field(
+        None,
+        description='The name of the data table that you want to '
+        'get (equivalent to "SELECT * FROM '
+        'your_table")',
+    )
     query: constr(min_length=1) = Field(
-        ..., description='You can write your SQL query here', widget='sql'
+        None,
+        description='You can write a custom query against your '
+        'database here. It will take precedence over '
+        'the "table" parameter above',
+        widget='sql',
     )
 
+    def __init__(self, **data):
+        super().__init__(**data)
+        query = data.get('query')
+        table = data.get('table')
+        if query is None and table is None:
+            raise ValueError("'query' or 'table' must be set")
+        elif query is None and table is not None:
+            self.query = f'select TOP 10 * from {table};'
+
+    @classmethod
+    def get_form(cls, connector: 'MSSQLConnector', current_config):
+        """
+        Method to retrieve the form with a current config
+        For example, once the connector is set,
+        - we are able to give suggestions for the `database` field
+        - if `database` is set, we are able to give suggestions for the `table` field
+        """
+        if current_config.get('database'):
+            connection = pyodbc.connect(
+                **connector.get_connection_params(database=current_config.get('database'))
```

Comment:
poruquoi pas une valeur par défaut ? 
```suggestion
                **connector.get_connection_params(database=current_config.get('database', 'tempdb'))
```